### ISSUE TITLE
AVT 2 - Accordion fails color contrast when the title is highlighted with a mouse

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4375]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React

## Detailed Description
Go to the React Accordion Component
Mouse over the accordion title
The title fails color contrast for WCAG 2.1 - 1.4.11 (see screenshot):
<img width="1181" alt="Screen Shot 2019-10-17 at 2 44 14 PM" src="https://user-images.githubusercontent.com/21676914/67042275-2b8bac00-f0ed-11e9-8101-d2885e25f112.png">

**Comment by dakahn** [comment]
@aagonzales My assumption is that this is a bug. What token should hover for accordion be using though?

**Comment by abbeyhrt** [comment]
Hey! @aagonzales would you be able to address the question above?

**Comment by aagonzales** [comment]
@snidersd which elements are you testing? The hover background (#e5e5e5) to the page background (#ffffff)? Why does that need to pass color contrast? I'm not aware of a rule where hover background have to pass, only the text. The text #161616 on the #e5e5e5 background passes at 14.37:1

**Comment by snidersd** [comment]
@aagonzales I just retested and I'm not seeing the issue with Color Contrast any longer. Please disregard. @dakahn Please close the issue. Just FYI... I was testing against 1.4.11 Non-text contrast, but I'm not able to reproduce the issue. Thx
